src/pages/User/User.js

```diff
@@ -44,27 +44,28 @@ class User extends React.Component {
   };
 
   render() {
-    const { data, value } = this.state;
+    const { data, value, username } = this.state;
     const isPhotos = value === 0;
     const isCollections = value === 1;
     const isStats = value === 2;
+    const readyToLoad = this.state.data && !this.state.isLoading;
     return (
       <>
         <LoadingBar color="#6958f2" ref={this.loadingBar} />
-        {this.state.data && !this.state.isLoading && (
+        {readyToLoad && (
           <>
             <img src={data.profile_image.large} alt={data.name} />
             <h1>{data.name}</h1>
-            {data.badge ? <Verified>Verified ✓</Verified> : ""}
+            {data.badge ? <Verified>Verified ✓</Verified> : null}
```

Comment:
data.badge && <Verified>Verified ✓</Verified> ? I dont think there's need for a ternary here...